### PR TITLE
Add isOpenBefore() to find the previous closing time 

### PR DESCRIPTION
**Pull request by flaktack**
This adds `OpeningHoursEvaluator#isOpenBefore` to find the closing time if it is currently closed.

It also add an explicit dependency on `ch.poole:OpeningHoursParser` since it is present in `mavenCentral()`.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -28,9 +28,7 @@ ext {
 }
 
 dependencies {
-    // by using the compile time classpath we add the parser classes to the jar but the dependency doesn't end up
-    // in the POM's <dependency> block
-    compileOnly group: 'ch.poole', name: 'OpeningHoursParser', version: parserVersion
+    implementation group: 'ch.poole', name: 'OpeningHoursParser', version: '0.23.2'
```
Comment (leonardehrenfried) [inline build.gradle:31]: Ok, is this on Maven Central now? If yes, that's great!

**Comment by leonardehrenfried** [comment]
I think you forgot to include your CSV file.

**Comment by flaktack** [comment]
Yes, I've added it :-)

File: build.gradle (hunk 1 above)
Comment (flaktack) [reply build.gradle:31]: I've also updated the readme to remove the relevant part.

**Comment by flaktack** [comment]
I've left it as draft, so that while I figure out how it to integrate with OTP it's easy to adjust.

**Comment by flaktack** [comment]
This contains a working implementation within OTP (https://github.com/realCity/OpenTripPlanner/commits/feature/vehicle-parking-opening-hours) -- if the changes here look good, could you please merge it and create a release?

**Comment by leonardehrenfried** [comment]
Is `isOpenBefore` the correct name? Not sure I would understand what it means?

Does it answer the question when the last closing time was?

**Comment by flaktack** [comment]
* `isOpenNext` search forward for the next "open" time (including the current time)
   current time if open, or the next opening time
* `isOpenBefore` searches backward for the previous "open" time (including the current time)
  current time of open, or the previous closing time

I agree it's not the best name, do you have any suggestions?

**Comment by flaktack** [comment]
Yes, that is significantly better, thanks! :-)

**Comment by leonardehrenfried** [comment]
I released 1.0.0. Give it a few minutes to arrive on Maven Central.

**Comment by flaktack** [comment]
Thanks! :partying_face:

**Pull request merged**